india_banking_connector/connectors/doctype/idfc_connector/idfc_connector.py

```
@@ -0,0 +1,415 @@
+		self.transaction_id = "NEFT" + unique_id
+
+		if existing_payment_response := self.validate_duplicate_payments(
+			unique_id=self.transaction_id,
+		):
+			return existing_payment_response
+
+		url = self.urls.make_payment
+		headers = self.headers
+		payload = self.get_encrypted_payload(method="make_payment")
+
+		response = requests.post(url, headers=headers, data=payload)
```

Ruff (0.15.0)

[error] Probable use of `requests` call without timeout

(S113)